projects/observability/src/shared/components/cartesian/cartesian-chart.component.scss

```diff
@@ -118,13 +119,41 @@
         }
 
         .legend-text {
-          fill: $gray-5;
           font-size: 14px;
           padding-left: 2px;
+          cursor: pointer;
+
+          &.default {
+            color: $gray-9;
+          }
+
+          &.active {
+            color: $blue-4;
+          }
+
+          &.inactive {
+            color: $gray-5;
+          }
         }
       }
     }
 
+    .reset {
```

Comment:
Is this for legend-text only? If so, can we place it inside `.legend`  or  `.legend-text?

Reply:
it is inside `.legend`